data/modifications/pick_advance/main.lua

```diff
@@ -198,15 +198,14 @@ on_event("start", function()
 end)
 
 -- set "fresh_turn" for the moveto event at the start of each side turn
-local fresh_turn = false
 on_event("turn refresh", function()
-	fresh_turn = true
+	wml.variables.pa_fresh_turn = true
 end)
 
 -- the first time a unit moves at the start of each side's turn, check if there are any new units that need to be forced to make an advancement choice
```

Comment:
why it needs to happen on movement at all?

Reply:
I have no idea